.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["esbenp.prettier-vscode"]
+}
```

Comment:
This is another change related to Prettier. You can ignore it. You can just mark this comment as resolved.